frontend/src/app/components/dashboard/dashboard.component.html

```
@@ -1,4 +1,4 @@
-<app-placeholder-table-view *ngIf="loading"></app-placeholder-table-view>
+<app-placeholder-table-view *ngIf="loading && !noTablesError && !isServerError"></app-placeholder-table-view>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)